src/utils/validateIdentifierInputs.ts

```diff
@@ -23,6 +25,15 @@ export const validateIdentifierInputs = (
     if (textInputType === "NUMBER" && isNaN(Number(value))) {
       throw new Error(ERROR_MESSAGE.INVALID_IDENTIFIER_INPUT);
     }
+    if (
+      textInputType === "PHONE_NUMBER" &&
+      !isMatchRegex(
+        value,
+        validationRegex ? validationRegex : defaultPhoneNumberValidationRegex
+      )
+    ) {
+      throw new Error(ERROR_MESSAGE.INVALID_PHONE_AND_COUNTRY_CODE);
+    }
```

Comment:
Oh wait did you add this or is this cuz we haven't rebase from master

Reply:
I didn't add this, we should rebase and then see what else we missed!